README.md

```diff
@@ -20,11 +20,11 @@ Hadoop.
   
   http://trafodion.apache.org/release-notes.html
 
-  - the latest Apache Trafodion can be downloaded from Apache incubator
+  - the latest Apache Trafodion can be downloaded from Apache
     distribution site or accessed from git:
 
     - download site:
-      https://dist.apache.org/repos/dist/release/incubator/trafodion
+      https://dist.apache.org/repos/dist/release/trafodion
```

Comment:
currently this URL cannot be found...

Reply:
Good point. I was anticipating this change.  I thought I could move the dist site with a couple svn commands, but I lack permission, so I have inquired with infrastructure team.

Reply:
Infra team has taken care of moving the dist directory.